<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectorSubCatsAnalysis</class>
 <widget class="QWidget" name="SectorSubCatsAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccione sector</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSelectSector">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxDays">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Días</string>
   </property>
   <widget class="QRadioButton" name="radioButtonWorkDay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lunes a viernes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSaturday">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sábado</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSunday">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Domingo</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxVisType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Tipo de visualización</string>
   </property>
   <widget class="QRadioButton" name="radioButtonAverageDisag">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Promedio desagregado</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAverageTotal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Promedio total</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
